tests/features/filters/filters.spec.ts

```diff
@@ -0,0 +1,154 @@
+          noResultsTitleText,
+        );
+        await pages.topPanel.clickTab('Catalog');
+        await pages.catalog.searchByTextInFilter('Owner', 'firstOwner');
+        expect(await pages.catalog.isListItemVisible(bookETLDataEntity)).toBeTruthy();
+      });
+    });
+    /**
+     * /project/1/test-cases/26
+     */
+    test('Should display the expected item with Tag filter', async ({ steps: { pages } }) => {
+      await test.step('Apply filter Tag', async () => {
+        await pages.catalog.searchByText(bookETLDataEntity);
+        await pages.catalog.clickOnListItem(bookETLDataEntity);
+        await pages.overview.createTag(tagFilterOption, noResultsTagText);
+        await pages.topPanel.clickTab('Catalog');
+        await pages.catalog.searchByTextInFilter('Tag', 'aqa_tag');
+        expect(await pages.catalog.isListItemVisible(bookETLDataEntity)).toBeTruthy();
+      });
+    });
+    /**
+     * /project/1/test-cases/70
+     */
+    test('Should display the expected item with Type filter', async ({ steps: { pages } }) => {
+      await test.step('I go to the Transformers tab', async () => {
+        await pages.catalog.clickTab(transformersTag);
+      });
+      await test.step('Apply filter Type', async () => {
+        await pages.catalog.openFilterWithInput(typeFilter);
+        await pages.catalog.searchByTextInFilter('Type', 'VIEW');
+        expect(await pages.catalog.isListItemVisible(bookETLDataEntity)).toBeTruthy();
+      });
+    });
+    /**
+     * /project/1/test-cases/71
+     */
+    test('Should display the expected item with Group filter', async ({ steps: { pages } }) => {
+      await test.step('Apply filter Group', async () => {
+        await pages.catalog.openFilterWithInput(groupsFilter);
+        await pages.catalog.chooseOption(groupsOption);
+        expect(await pages.catalog.isListItemVisible(bookETLDataEntity)).toBeTruthy();
+      });
+    });
+  });
+  test.describe('When apply multiple filters', () => {
+    test.beforeEach(async ({ steps: { pages }, page }) => {
+      await test.step(`I open catalog page`, async () => {
+        await page.goto('');
+        await pages.topPanel.clickTab('Catalog');
+      });
+    });
+    /**
+     * /project/1/test-cases/28
+     */
+    test('Should display the expected item with different types of filters', async ({
+      steps: { pages },
+    }) => {
+      await test.step('Apply filter Tag', async () => {
+        await pages.catalog.openFilterWithSelect(namespaceFilter);
+        await pages.catalog.chooseOption(namespaceOption);
+        expect(await pages.catalog.isListItemVisible(bookETLDataEntity)).toBeTruthy();
+      });
+      await test.step('Apply filter Owner', async () => {
+        await pages.catalog.openFilterWithInput(ownerFilter);
+        await pages.catalog.chooseOption(firstOwner);
+        expect(await pages.catalog.isListItemVisible(bookETLDataEntity)).toBeTruthy();
+      });
+    });
+    /**
+     * /project/1/test-cases/28
```

Comment:
same testId as for the previous test, please check Id and proper summary and steps